From 684a81e546731562211dedbf40c1a118037bcb90 Mon Sep 17 00:00:00 2001
From: Martijn Braam <martijn@brixit.nl>
Date: Tue, 22 Dec 2020 16:05:06 +0100
Subject: [PATCH 1/2] Fix libtiff 4.2 incompatability

---
 process_pipeline.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/process_pipeline.c b/process_pipeline.c
index 33cb72b..7850c14 100644
--- a/process_pipeline.c
+++ b/process_pipeline.c
@@ -236,7 +236,11 @@ process_image_for_capture(const MPImage *image, int count)
 	TIFFSetField(tif, TIFFTAG_PLANARCONFIG, PLANARCONFIG_CONTIG);
 	static const short cfapatterndim[] = { 2, 2 };
 	TIFFSetField(tif, TIFFTAG_CFAREPEATPATTERNDIM, cfapatterndim);
+#if (TIFFLIB_VERSION < 20201219)
 	TIFFSetField(tif, TIFFTAG_CFAPATTERN, "\002\001\001\000"); // BGGR
+#else
+	TIFFSetField(tif, TIFFTAG_CFAPATTERN, 4, "\002\001\001\000"); // BGGR
+#endif
 	if (camera->whitelevel) {
 		TIFFSetField(tif, TIFFTAG_WHITELEVEL, 1, &camera->whitelevel);
 	}
-- 
2.29.2

